prac_05/word_occurences.py

```diff
@@ -0,0 +1,35 @@
+from string import punctuation
+# Prompt user for string
+text_string = input("Enter a sentence or paragraph: ")
```

Comment:
Could force string to be lower case for more accurate sorting (I didn't do this either).